pkg/client/client.go

```diff
@@ -666,7 +666,7 @@ func (c *Client) BuildProviderTables(ctx context.Context, providerName string) (
 		}
 
 		c.Logger.Error("BuildProviderTables failed", "error", retErr)
-		retErr = fmt.Errorf("Incompatible provider schema: Please drop provider tables and recreate, alternatively execute `cloudquery provider drop %s`", providerName)
+		retErr = fmt.Errorf("incompatible provider schema: Please drop provider tables and recreate, alternatively execute `cloudquery provider drop %s`", providerName)
```

Comment:
This was done (casing) deliberately as this error message is not internal and it's what the user sees.

Reply:
Ahh. Ok, that makes sense I wonder if there is a way to let the gofmt know it is on purposes because I was getting errors that it was "wrong"

Reply:
maybe something like `//nolint` ?